ui/.gitignore

```diff
@@ -13,3 +13,13 @@ src/static/env.js
 coverage
 test/e2e/cypress/screenshots/
 test/e2e/cypress/videos/
+
+
+# yarn package manager
+.yarn/*
```

Comment:
I'm wondering if we really need this change. Can you please tell how you get there? 
We've been using both `ui` and top-level services with `yarn` but we never had any files created in `.yarn` folder other than [.yarn/releases/yarn-1.22.19.cjs](https://github.com/taskcluster/taskcluster/blob/main/.yarn/releases/yarn-1.22.19.cjs) that we have on the top level.

Maybe you have some custom yarn configuration in your local dev environment? 
Thanks!

Reply:
Yeah you are right - I don't think its necessary. 

For some reason running 'yarn' was installing the .yarn/cache directory in the /ui project root but i think i fixed it.   My first time using yarn & nvm.

Reply:
Cool! Let's then revert those changes for this file.
Thank you

Reply:
Cool - i made another commit to revert the changes to the .gitignore  Hope that's ok

Reply:
Awesome! Thanks again!